js/reference.js

```diff
@@ -3258,7 +3262,15 @@ var ERMrest = (function(module) {
             }
             return {isHTML: isHTML, value: value};
         },
-
+        /**
+         * Append a <pre> tag to the value so that the beautified JSON can be shown including the whitespaces.
+         * @param {String} value This is the formatted string that we get after stringifying
+         * @returns {String} A string representation of the stringified value enclosed in a <pre> tag
+         */
+        appendPreTag: function appendPreTag(value) {
```

Comment:
You've just used this function one time, and also this function doesn't make any sense for column. Remove it and just append the text instead of calling a function.

Remember whatever apis that you are exposing here might be used by client (chaise), so if the api that you are creating doesn't make sense to client you shouldn't expose it.